design/PPE/ippe_vibrate_outlined.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M12 17.7L6.3 12L12 6.30002L17.7 12L12 17.7ZM19.4 10.9L13.7 5.20002C12.7 4.20002 11.1 4.20002 10.2 5.20002L4.5 10.9C3.9 11.5 3.9 12.4 4.5 13L10.2 18.7C11.2 19.7 12.8 19.7 13.7 18.7L19.4 13C20 12.5 20 11.5 19.4 10.9ZM13.4 20.5L14.8 21.9L21.9 14.8L20.5 13.4L13.4 20.5ZM21.2 16.9L17 21.1L18.4 22.5L22.6 18.3L21.2 16.9ZM2.1 9.20002L3.5 10.6L10.6 3.50002L9.2 2.10002L2.1 9.20002ZM1.4 5.60002L2.8 7.00002L7 2.80002L5.6 1.40002L1.4 5.60002Z" />
```

Comment:
material has vibration https://px-blue.github.io/style/iconography?icon=Vibration 

don't know what's the significance of tilting it

Reply:
This is specifically representing the iPPE device